biolink-model.yaml

```diff
@@ -1148,7 +1168,7 @@ slots:
     in_subset:
       - translator_minimal
     mappings:
-      - WD:P2293
+      - WD:Property:P2293
```

Comment:
Are double colons the only option?

is the intent here to change the local-identifier to  "Property:P2293"  
while keeping the curie-prefix valid as "WD"  
or is it to keep the original local-identifier "P2293" 
and promote the invalid curie-prefix "WD:Property" ?

Reply:
Hi @TomConlin, this is a Wikidata quirk: https://www.wikidata.org/wiki/Property:P2293 is the URI for property P2293.  See for yourself... click on the link.  I tried to capture this with the embedded colon.  Broke the build. 

Regular Wikidata entity entries resolve to simpler URIs with 'Q' numbers, e.g. https://www.wikidata.org/wiki/Q1154619

I'm OK with using the prefix 'wikidata' for these (there are some places which express that prefix preference) but this won't automatically resolve Properties (i.e. https://www.wikidata.org/wiki/P2293 doesn't resolve to the said property). 

Maybe we don't care about that.. the properties all start with 'P' after all? And, this would not be our only non-standard base URI in the model.

To fix the Travis build of the model, I just reverted to using `WDP:` prefix set to `https://www.wikidata.org/wiki/Property:` which seems to allow the build to pass.... but perhaps this is unacceptable?

All that said, as far I can see, there are only two (2) Wikidata CURIEs left in the model file (used to be several more.. not sure what happened to them). @cmungall do we really even need to reference Wikidata here (in Biolink)?

Reply:
the spec also leaves a loophole you can drive through with 
talking about what comes before and after the  "... first colon ..."

Someone (possibly me) did this

https://github.com/monarch-initiative/dipper/blob/master/dipper/curie_map.yaml#L256

Reply:
Hi @TomConlin, I actually did something similar (in the latest, cleaned up version of the commit to this PR):

```  
  WIKIDATA: 'https://www.wikidata.org/wiki/'
  WIKIDATA_PROPERTY: 'https://www.wikidata.org/wiki/Property:'
```

The `WIKIDATA:` prefix is pretty conventional in some quarters. The `WIKIDATA_PROPERTY:` is not, but both clear BiolinkML and seem to be a practical workaround.